oeis.py

```diff
@@ -139,6 +138,22 @@ def A000045(start: int = 0, limit: int = 20) -> Collection[int]:
         sequence.append(sequence[i - 1] + sequence[i - 2])
     return sequence
 
+@oeis
+def A000566(start: int = 0, limit: int = 20) -> Collection[int]:
+    def aList():
```

Comment:
I don't like this function name, also PEP8 tells it should use snake_case not camelCase.

Maybe:
```suggestion
    def A000566_generator():
```

Reply:
Oui vous aviez raison ... Désolé